snoop/html.py

```diff
@@ -30,32 +30,29 @@ def _extract_links(html):
     return html
 
 def _create_lxml_html_cleaner():
+    # Consult http://lxml.de/3.4/api/lxml.html.clean.Cleaner-class.html
     cleaner = lxml.html.clean.Cleaner()
 
-    # Only the tags will be removed, their content will get pulled up into
-    # the parent tag. The idea here is to make it explicit to copy/paste a link
-    # Also remove any tags that would make inline printing bad (html, head, body)
+    # The idea here is to make it explicit to copy/paste a link, as all of our
+    # links will be available as a text-only index.
+    # Also remove any tags that would make inline printing bad, like <html>,
```

Comment:
There might be stylesheet code outside `<head>`, i do that sometimes when i'm lazy, and browsers will happily use it. So maybe remove all `<style>` and `<link>` tags, regardless of where they appear in the page?